pytorch_toolkit/image_classification/model_templates/custom-classification/mobilenet_v2_w1/template.yaml

```diff
@@ -0,0 +1,58 @@
+name: mobilenet-v2-w1-imagenet
+domain: Image classification
+problem: ImageNet classification
+framework: OTEReidentification v1.2.3
+summary: Imagenet classification based on MobileNet-V2.
+annotation_format: CustomImages
```

Comment:
What is it?

Reply:
It's actually just a classical format which supposes each class to be represented by a folder. May be I should have called it ImageFolder like in torchvision or indeed add support of COCO.